couchbase-operator/generate.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Helper script to automatically update the values.yaml and README.md from the CRD.
+#
+set -eux
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# Defaults to everything required locally but useful for testing to have configuration options
+CRD_FILE=${CRD_FILE:-$SCRIPT_DIR/crds/couchbase.crds.yaml}
+CHART_DIR=${CHART_DIR:-$SCRIPT_DIR}
+OUTPUT_DIR=${OUTPUT_DIR:-$SCRIPT_DIR}
+OUTPUT_VALUES_FILE=${OUTPUT_VALUES_FILE:-$OUTPUT_DIR/values.yaml}
+OUTPUT_README_FILE=${OUTPUT_README_FILE:-$OUTPUT_DIR/README.md}
+
+# First add the manually controlled source
+cat "${SCRIPT_DIR}/values.yamltmpl" > "${OUTPUT_VALUES_FILE}"
+# Now autogenerate the rest of the values file
+CRD_FILE=${CRD_FILE} /bin/bash "${SCRIPT_DIR}/../tools/value-generation/generateValuesFile.sh" >> "${OUTPUT_VALUES_FILE}"
```

Comment:
Need to run docker silent here (or redirect its output), otherwise the build steps get appended to values file.

Reply:
It works fine for me on MacOS as I redirect the `docker run` to the values file inside that script.